test/Resolver.t.sol

```diff
@@ -0,0 +1,138 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.25;
+
+import { Test, console2 } from "forge-std/src/Test.sol";
+import { Resolver } from "../src/resolver/Resolver.sol";
+import { IResolver } from "../src/interfaces/IResolver.sol";
+import { ISchemaRegistry } from "../src/interfaces/ISchemaRegistry.sol";
+import { IEAS } from "../src/interfaces/IEAS.sol";
+import { TestMockedResolver } from "../src/resolver/TestMockedResolver.sol";
+
+contract ResolverTest is Test {
+  IEAS eas = IEAS(0xC2679fBD37d54388Ce493F1DB75320D236e1815e);
+  ISchemaRegistry schemaRegistry = ISchemaRegistry(0x0a7E2Ff54e76B8E6659aedc9103FB21c038050D0);
+  TestMockedResolver testMockedResolver;
+  
+
+    bytes32 public schemaId;
+    address deployer = 0xF977814e90dA44bFA03b6295A0616a897441aceC; //TODO: change address
+
+    function setUp() public {
+        vm.label(deployer, "deployer");
+        vm.startPrank(deployer);
+        testMockedResolver = new TestMockedResolver(eas);
+        schemaId = schemaRegistry.register("bool isVerified", address(testMockedResolver), true);
+    }
+
+  function test_mocked_attest() public {
+        vm.stopPrank();
```

Comment:
You don't need to stop a prank before starting a new one. The new prank will overwrite the older.